scripts/zkai_attention_kv_stwo_fusion_mechanism_ablation_gate.py

```
@@ -0,0 +1,664 @@
+    "binary_accounting_overclaim",
+    "non_claim_removed",
+    "evidence_sha_drift",
+    "unknown_field_injection",
+]
+
+
+class FusionMechanismAblationGateError(ValueError):
+    pass
+
+
+def canonical_json_bytes(value: Any) -> bytes:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)